ibis/pyspark/compiler.py

```diff
@@ -85,7 +87,8 @@ def compile_selection(t, expr, scope, **kwargs):
     for selection in op.selections:
         if isinstance(selection, types.TableExpr):
             col_names_in_selection_order.extend(selection.columns)
-        elif isinstance(selection, types.ColumnExpr):
+        elif (isinstance(selection, types.ColumnExpr)
```

Comment:
`isinstance(selection, (types.ColumnExpr, types.ScalarExpr)`
?

Reply:
Is `ScalarExpr` basically literal?

Reply:
Yep, this should allow for expressions like:

```
expr = ibis.now()
result = con.execute(expr)
```